### PR TITLE
fix(amplify-category-function): Storage env vars not added to lambda function

### LINKED ISSUE #7718: Storage env vars not added to lambda function
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

node 14

### Amplify CLI Version

5.1.1

### What operating system are you using?

mac

### Amplify Categories

storage, function

### Amplify Commands

add

### Describe the bug

1. When adding a lambda trigger that accesses a table and no resources, the environment variables for ENV, REGION, and TABLE_NAME show up in both the CLI and get printed for reference in the lambda itself.

When doing the same thing, but additionally selecting storage as a resource one would like access to, those environment variables are not added to the generated lambda function.

### Expected behavior

I would expect the STORAGE_BUCKETNAME_BUCKET env var to be printed along with all the other env vars.

### Reproduction steps

In this blog post: https://dev.to/mtliendo/create-a-resume-uploader-using-aws-amplify-2e17

*Add auth
* add an api
* add storage
* add a function

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line
type Candidate 
  @model 
  @auth(rules: [{ allow: owner, operations: [create, update] }]) {
    id: ID!
    name: String!
    email: String!
    resumeFile: String!
    userIdentity: String!
}

```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by sachscode**
#### Description of Bug Fix
After this bug fix the file header with environment variables is generated in the lambda examples. 

* Enabled EJS templates for Lambda function trigger flow.
* Added feature to display environment variables on the screen after completion of step.
Screen Shot 2021-07-22 at 11 31 49 PM

* Fixed code to update top-level header after environment variables are added.

Screen Shot 2021-07-22 at 11 20 08 PM

#### Issue 
Storage env vars not added to lambda function 
Fixes #7718 
#### Description of how you validated changes
- manually tested steps in https://dev.to/mtliendo/create-a-resume-uploader-using-aws-amplify-2e17
- Unit tests.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-nodejs-function-template-provider/src/utils/analyticsWalkthrough.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export async function askAnalyticsCategoryKinesisQuestions(context: any) {
         batchSize: 100,
         startingPosition: 'LATEST',
         eventSourceArn: streamArnParamRef,
-        functionTemplateName: 'trigger-kinesis.js',
+        functionTemplateName: 'trigger-kinesis.js.ejs',
```
Comment (ammarkarachi) [inline packages/amplify-nodejs-function-template-provider/src/utils/analyticsWalkthrough.ts:40]: Does it not render with props if `.ejs` is not specified?

Comment (sachscode) [reply packages/amplify-nodejs-function-template-provider/src/utils/analyticsWalkthrough.ts:40]: It does not. The file itself was js , for the trigger flow. Hence the ejs.render() function was not getting called for this file.

**Review by edwardfoyle — APPROVED** [review]

File: packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.test.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { topLevelCommentPrefix, topLevelCommentSuffix, envVarPrintoutPrefix } from '../../../../constants';
```
Comment (jhockett) [inline packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.test.ts:1]: Please use 2 spaces for indentation.

File: packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.test.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            },
+            "SES_EMAIL": {
+              "Ref": "sesEmail"
+            }
+        }
+        const outputString = `${envVarPrintoutPrefix}ENV\n\tREGION\n\tSTORAGE_MOCK_BUCKETNAME\n\tSES_EMAIL`;
+        expect( buildShowEnvVars( inputEnvMap ) ).toEqual(outputString);     
+    });
+    
+});
```
Comment (jhockett) [inline packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.test.ts:46]: nit: add a new line to the end of the file

**Review by jhockett — APPROVED** [review]
2 nits, but lgtm

**Pull request merged**